package/gluon-client-bridge/luasrc/lib/gluon/upgrade/300-gluon-client-bridge-network

```diff
@@ -40,31 +40,41 @@ uci:section('network', 'interface', 'client', {
 
 uci:save('network')
 
-
+-- TODO: remove this line and the next in 2019. Firewall zones have been renamed in 2017.
 uci:delete('firewall', 'client')
-uci:section('firewall', 'zone', 'client', {
-	name = 'client',
+
+uci:section('firewall', 'zone', 'drop', {
+	name = 'drop',
 	network = {'client'},
```

Comment:
This commit also needs to adjust `/lib/gluon/upgrade/320-gluon-mesh-batman-adv-client-bridge` in the gluon-mesh-batman-adv package. In particular, the client interface must be moved from the *drop* to the *mesh* zone there.

Reply:
I did multiple things here reworking this portion of the PR:
- remove the superseeding policy settings in gluon-mesh-batman-adv for the mesh-zone. this used to be ACCEPT for inbound. Now the base policy is used having REJECT for INBOUND
- add the client network to the mesh zone
- remove the client network from the drop zone.